ui/app/services/permissions.js

```diff
@@ -86,24 +97,36 @@ export default Service.extend({
     } catch (err) {
       // If no policy can be found, default to showing all nav items.
       this.set('canViewAll', true);
-      this.set('readFailed', true);
+      this.set('permissionsBanner', PERMISSIONS_BANNER_STATES.readFailed);
     }
   }),
 
+  calcNsAccess() {
+    if (this.canViewAll) {
+      this.set('permissionsBanner', null);
+      return;
+    }
+    const namespace = this.baseNs;
+    const allowed =
+      Object.keys(this.globPaths).any((k) => k.startsWith(namespace)) ||
```

Comment:
For someone not super familiar with this response, could you maybe add a comment with an example of glob vs exact paths? Especially since the API doc are 🙃 in information for this endpoint. Found [this](https://stackoverflow.com/questions/70234255/how-to-identify-what-paths-can-be-accessed-with-what-capabilities-in-hashicorp-v) stackoverflow with someones examples to help me visual it.

Reply:
I think the best example is in the [test file](https://github.com/hashicorp/vault/blob/main/ui/tests/unit/services/permissions-test.js)

I'd love to make this service glimmer + typescript, but since this change will be backported that will have to be a follow on